### ISSUE TITLE
Lint Angular class attribute name bindings

### ISSUE THREAD

**Issue by JasonWeinzierl** [schoero/eslint-plugin-readable-tailwind#274]
### Description

The [Angular documentation suggests](https://angular.dev/guide/templates/binding#css-classes) using direct attribute binding for CSS classes (e.g. `[class.text-bg-primary]="true"`), which doesn't seem to get parsed by this plugin.  My app is open-source, so I've linked the entire repo if that helps.

### Flavor

Angular

### Code Input

```html
<span
  class="badge"
  [class.text-bg-primry]="banner.isEnabled" <-- typo in attribute
  [class.text-bg-secondary]="!banner.isEnabled"
>
  {{ banner.isEnabled ? 'Visible' : 'Hidden' }}
</span>
```

### Expected Behavior

A typo e.g. `[class.text-bg-primry]` should get flagged by `no-unregistered-classes`.

### Actual Behavior

No error occurs.

### Reproduction URL

Above example is modified from my repo here:

https://github.com/JasonWeinzierl/BHS.NET/blob/main/src/bhs-web-angular-app/src/app/features/admin/pages/admin-banners/admin-banners.component.html#L27

### ESLint Log

```shell
(No output)
```

### ESLint Config

https://github.com/JasonWeinzierl/BHS.NET/blob/main/src/bhs-web-angular-app/eslint.config.mjs#L323

```js
{
  plugins: {
    'better-tailwindcss': tailwindcss,
  },
  settings: {
    'better-tailwindcss': {
      entryPoint: 'src/styles.css',
    },
  },
  rules: {
    ...tailwindcss.configs['recommended-error'].rules,

    'better-tailwindcss/no-unregistered-classes': [
      'error',
      {
        ignore: [
          // Bootstrap icons.
          '^bi$',
          '^bi-',
        ],
      },
    ],
  },
}
```

### Versions

- ESLint: ^9.39.2
- Parser:     "angular-eslint": "21.1.0",
- Plugin:     "eslint-plugin-better-tailwindcss": "^3.8.0",
- Node: v24.12.0
